ecosystem/sep-0024.md

```diff
@@ -193,9 +192,12 @@ The withdraw endpoint allows a wallet to get withdrawal information from an anch
 ### Request
 
 ```
-GET TRANSFER_SERVER/withdraw
+POST TRANSFER_SERVER/withdraw
+Content-Type: multipart/form-data
```

Comment:
We haven't really settled on anything.  SEP-9 has some binary fields which would require form-data, but I kinda doubt anyone will ever use them.  I considered not defining and letting wallets pass either type (with the appropriate content-type), but using form-data here was just me following SEP-9's lead

Reply:
Binary fields seem pretty important for accepting an upload of photo identification. Using form-data for that reason makes sense if we need to bundle file uploads with providing information, but we should separate the two if it's not too late to do so. The anchor can provide an endpoint for uploading a document, and then this API supports providing the identifier of that document.

A good example of this type of API behavior is Stripe's APIs:
https://stripe.com/docs/file-upload

If we break it out, we might also want to think about how we could allow an anchor to use the common direct to storage upload features of AWS S3 / Google Cloud Storage for sensitive document upload.

Reply:
>Other than direct-to-s3 support, what are the benefits of breaking binary
into its own request?  Would SEP-12 fulfill that?

@msfeldstein I forgot about SEP-12, yup it solves that problem. It might still be worth us taking a look at the S3 and similar APIs and seeing if we could reduce the format of what we post down to be compatible, and making it so that the KYC/Transfer server is asked for a URL to upload a file to, instead of having one common URL that everyone uploads files to. I think that should make it compatible with those APIs then.

Reply:
I'm going to resolve this.  I think wallets and anchors can use SEP-12 if they want the drivers license image ahead of time, that seems like such an edge case we don't need to advertise it here.  I also don't know if we really want to encourage wallets to collect that level of personal data anyway.